kentikapi/internal/api_resources/my_kentik_portal_test.go

```diff
@@ -0,0 +1,228 @@
+	myKentikPortalAPI := api_resources.NewMyKentikPortalAPI(transport)
+
+	//act
+	tenant, err := myKentikPortalAPI.Get(nil, 577)
+
+	//assert
+	assert := assert.New(t)
+	require := require.New(t)
+
+	require.NoError(err)
+	assert.Zero(transport.RequestBody)
+
+	assert.Equal(models.ID(577), tenant.ID)
+	assert.Equal("test_tenant", tenant.Name)
+	assert.Equal("This is test tenant", tenant.Description)
+	assert.Len(tenant.Users, 2)
+	assert.Equal(2020, tenant.CreatedDate.Year())
+	assert.Equal(time.Month(12), tenant.CreatedDate.Month())
+	assert.Equal(21, tenant.CreatedDate.Day())
+	assert.Equal(10, tenant.CreatedDate.Hour())
+	assert.Equal(55, tenant.CreatedDate.Minute())
+	assert.Equal(52, tenant.CreatedDate.Second())
+	assert.Equal(2020, tenant.UpdatedDate.Year())
+	assert.Equal(time.Month(12), tenant.UpdatedDate.Month())
+	assert.Equal(21, tenant.UpdatedDate.Day())
+	assert.Equal(10, tenant.UpdatedDate.Hour())
+	assert.Equal(55, tenant.UpdatedDate.Minute())
+	assert.Equal(52, tenant.UpdatedDate.Second())
+	assert.Equal(models.ID(148099), tenant.Users[0].ID)
+	assert.Equal("test@tenant.user", tenant.Users[0].Email)
+	assert.Nil(tenant.Users[0].LastLogin)
+	assert.Equal(models.ID(577), tenant.Users[0].TenantID)
+	assert.Equal(models.ID(74333), tenant.Users[0].CompanyID)
+	assert.Equal(models.ID(148113), tenant.Users[1].ID)
+	assert.Equal("user@testtenant.com", tenant.Users[1].Email)
+	assert.Nil(tenant.Users[1].LastLogin)
+	assert.Equal(models.ID(577), tenant.Users[1].TenantID)
+	assert.Equal(models.ID(74333), tenant.Users[1].CompanyID)
+}
+
+func TestTenantUserCreate(t *testing.T) {
+	createTenantUserResponse := `
+	{
+		"id": "158564",
+		"user_email": "test@test.test",
+		"last_login": null,
+		"tenant_id": "578",
+		"company_id": "74333"
+	}`
+
+	transport := &api_connection.StubTransport{ResponseBody: createTenantUserResponse}
+	myKentikPortalAPI := api_resources.NewMyKentikPortalAPI(transport)
+
+	//act
+	tenantUser, err := myKentikPortalAPI.CreateTenantUser(nil, 577, "test@test.test")
+
+	//assert
+	assert := assert.New(t)
+	require := require.New(t)
+
+	require.NoError(err)
+	payload := utils.NewJSONPayloadInspector(t, transport.RequestBody)
+	assert.NotNil(payload.Get("user"))
+	assert.Equal("test@test.test", payload.String("user/user_email"))
+
+	assert.Equal(models.ID(158564), tenantUser.ID)
+	assert.Equal("test@test.test", tenantUser.Email)
+	assert.Nil(tenantUser.LastLogin)
+	assert.Equal(models.ID(578), tenantUser.TenantID)
+	assert.Equal(models.ID(74333), tenantUser.CompanyID)
+
+}
+
+func TestTenantUserDelete(t *testing.T) {
+	// arrange
+	deleteResponsePayload := ""
+	transport := &api_connection.StubTransport{ResponseBody: deleteResponsePayload}
+	myKentikPortalAPI := api_resources.NewMyKentikPortalAPI(transport)
+
+	// act
+	tenantID := models.ID(478)
+	userID := models.ID(420)
+	err := myKentikPortalAPI.DeleteTenantUser(nil, tenantID, userID)
+
+	// assert
+	assert := assert.New(t)
```

Comment:
There is no need to assertion objects in such tests.
You could just use `assert.Zero(t, foo)` forms, etc.